chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/NonconformityType.java

```diff
@@ -51,6 +51,10 @@ public void setId(Long id) {
         this.id = id;
     }
 
+    public CertificationEdition getCertificationEdition() {
```

Comment:
Does adding this element change the XSD? If it does, we'll need the XSD-Impact label and a change to the XSD-MODIFICATION file

Reply:
It does not look to be in the XSD.
![image](https://user-images.githubusercontent.com/24817202/217007822-6951b452-ba34-448e-8e3a-5efa88bc558a.png)